kedro/framework/session/session.py

```diff
@@ -93,16 +94,19 @@ class KedroSession:
 
     """
 
+    # pylint: disable=too-many-arguments
     def __init__(
         self,
         session_id: str,
         package_name: str = None,
         project_path: Union[Path, str] = None,
         save_on_close: bool = False,
+        conf_source: Optional[str] = None,
     ):
         self._project_path = Path(project_path or Path.cwd()).resolve()
         self.session_id = session_id
         self.save_on_close = save_on_close
+        self._conf_source = conf_source
```

Comment:
```suggestion
```
No point of this if you are reassigning it again later.

Reply:
Yes! Thanks for the catch, this was a leftover from an earlier implementation that I missed. Fixed in [fde701c](https://github.com/kedro-org/kedro/pull/2117/commits/fde701c1ed0cffa5ecf5f340291ecb1267470caa).